### ISSUE TITLE
clap assertion failure in cli.yml

### ISSUE THREAD

**Issue by edgex004** [tauri-apps/tauri#2506]
### **Describe the bug**

Clap throws an assertion error when running `tauri build`.

Side note: it seems like `load_yaml!("cli.yml");` does not mark `main.rs` as dirty when `cli.yml` changes. I need to clean build to get the changes in `cli.yml` to be utilized.

### **To Reproduce**
1. cd to rs.cli directory.
2. Clean build rs.cli: `cargo clean && cargo build` 
2. cd to your project directory. (In this case, project was created with `npm x create-tauri-app`)
3. Build project with rs.cli executable: `path/to/cargo-tauri tauri build`
4. Note error: 
```
thread 'main' panicked at '
ArgSettings::TakesValue is required when ArgSettings::MultipleValues is set.
```

### **Expected behavior**
Project is built successfully.

### **Platform and Versions (required):**
```
Operating System - Mac OS, version 11.2.3 X64

Node.js environment
  Node.js - 16.7.0
  @tauri-apps/cli - 1.0.0-beta.9
  @tauri-apps/api - 1.0.0-beta.6

Global packages
  npm - 7.20.3
  yarn - 1.22.11

Rust environment
  rustc - 1.56.0-nightly
  cargo - 1.56.0-nightly

App directory structure
/dist
/target
/node_modules
/src-tauri
/.idea
/src

App
  tauri.rs - 1.0.0-beta.6 (no lockfile)
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../dist
  devPath - http://localhost:10001/
  bundler - Rollup
```

### **Additional context**
Built from hash `8fcdc1d47419f8f609b5685d5266d7c5365a7106`

**Comment by edgex004** [comment]
I have updated cli.yml to fix on my system. Will submit a PR.

**Comment by edgex004** [comment]
Would it be a good idea to add a test for cli.yml?

**Comment by edgex004** [comment]
Fixed by #2507.
